app/src/main/res/drawable/thumb_weclapp.xml

```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="670.6535dp"
```

Comment:
As per Android Studio considerations for SVGs it is recommended to limit a vector image to a maximum of 200 x 200 dp; otherwise, it can take too long to draw. See: https://developer.android.com/studio/write/vector-asset-studio#when

Reply:
I reduced the dimensions.